controllers/gitrepository_controller_test.go

```diff
@@ -637,6 +637,60 @@ func TestGitRepositoryReconciler_reconcileArtifact(t *testing.T) {
 				*conditions.TrueCondition(meta.ReadyCondition, meta.SucceededReason, "Stored artifact for revision 'main/revision'"),
 			},
 		},
+		{
+			name: "Remove unavailable condition after creating artifact",
```

Comment:
```suggestion
			name: "Removes ArtifactUnavailableCondition after creating artifact",
```

to be able to distinguish an "unavailble condition" from the type :-)